<launch>
    <arg name="is_sim" value="1" />
    <arg name="real_time" value="0"/>
    <arg name="target_timestep" value="1.0"/>

    <rosparam file="$(find gtddp_drone)/launch/constants/pursuit_sim_constants.yaml" command="load" />

    <!-- Debugging -->
    <param name="debug_mode" value="true" />

    <!-- GT-DDP nodes -->
    <node name="pursuit_control_node" pkg="gtddp_drone" type="pursuit_control_node" output="screen">
        <param name="is_simulation" value="$(arg is_sim)"/>
    </node>
    <node name="gtddp_optimize_node" pkg="gtddp_drone" type="gtddp_optimize_node" output="screen">
        <param name="is_simulation" value="$(arg is_sim)"/>
        <param name="is_real_time" value="$(arg real_time)"/>
    </node>

    <!-- Simulator -->
    <include file="$(find cvg_sim_gazebo)/launch/pursuit_emptyworld.launch"/>

    <!-- Keyboard node -->
    <node pkg="gtddp_drone" type="keyboard_node.py" name="keyboard_node" output="screen"/>

</launch>
